yb-voyager/src/tgtdb/suites/oracleSuite.go

```diff
@@ -44,6 +44,23 @@ var OraValueConverterSuite = map[string]ConverterFn{
 		}
 		return formattedDate, nil
 	},
+	"io.debezium.time.Date": func(columnValue string, formatIfRequired bool) (string, error) {
+		epochDays, err := strconv.ParseUint(columnValue, 10, 64)
```

Comment:
I don't think it would be right to assume that it's an unsigned int. 
> `fmt.Printf("date=%s\n", time.Unix(int64(-2), 0).UTC().Format(time.DateOnly))`

date=1969-12-31

pls change it to ParseInt (even for yb suite)

Reply:
@shubham-yb we should test dates prior to epoch as well 😅